conda/gateways/connection/adapters/ftp.py

```diff
@@ -289,8 +289,8 @@ def parse_multipart_files(request):
 
 
 def get_status_code_from_code_response(code):
-    """
-    The idea is to handle complicated code response (even multi lines).
+    r"""Handle complicated code response, even multi-lines.
```

Comment:
Why a raw string?

Reply:
Because there's an escaped character in the docstring (e.g., "\n"), this was linter suggested